pkg/kubelet/apis/device-plugin/v1alpha1/constants.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deviceplugin
+
+const (
+	// Healthy means that the device is healty
+	Healty = "Healthy"
+	// UnHealthy means that the device is unhealty
+	Unhealthy = "Unhealthy"
+
+	// Version is the API version
+	Version = "0.1"
+	// DevicePluginPath is the folder the Device Plugin is expecting sockets to be on
+	// Only privileged pods have access to this path
+	// Note: Placeholder until we find a "standard path"
+	DevicePluginPath = "/var/run/kubernetes/"
```

Comment:
FYI, just checked with cri folks. A path under /var/lib/ seems better because /home/kubernetes is mostly for staging. I guess device plugin unix socks can live under /var/lib/device-plugins/ and kubelet side device-plugin registration sock and checkpoint data can live under /var/lib/kubelet/device-plugin/?